rust/blockstore/src/arrow/config.rs

```diff
@@ -7,7 +7,8 @@ pub const TEST_MAX_BLOCK_SIZE_BYTES: usize = 16384;
 #[derive(Deserialize, Debug, Clone)]
 pub struct ArrowBlockfileProviderConfig {
     pub block_manager_config: BlockManagerConfig,
-    pub sparse_index_manager_config: SparseIndexManagerConfig,
+    #[serde(alias = "sparse_index_manager_config")]
```

Comment:
backwards compat

Reply:
I don't know enough about our deployment process. Is it possible to have new code reading old config or vice-versa?

Reply:
Yeah, thats why I alias, so that old config works